README.md

```diff
@@ -111,33 +111,20 @@ Then, Select the Python binary as `venv` in VSCode:
 
 ## Test The Util
 
-Make a new file like:
+Go to `tests/static_data` directory and create a file called "API_KEY" and 
+paste your API Key in this file. 
 
-`code test.py`
```

Comment:
I would actually keep this.  This is how we run local tests of the sdk (not unit tests).  Let's create a separate section for running unit tests.

Reply:
I would like to disagree here, developers should run unit tests as a habit, moreover the overhead is same (which would be to create an API_KEY file and then instead of python test.py run python -m tests\Basic_tests.py). I am happy to keep it here but eventually it would be about moving to a robust testing for developers rather than just using single test or inconsistent testing by different devs.

Reply:
Completely agree on unit tests.  I guess I just want to leave a basic usage of the SDK in the README (like a quick start guide on how to make simple declarations and calls).  This shouldn't be called test, it's confusing.  

We still need a way to run the sdk with real data (using a live key) , so we can troubleshoot and/or support any issues our partner might run into.  Does that make sense @tanmaydattta ?

Reply:
Yes and that API_KEY should be able to model some part. We need to have two kind of tests. 
1. Unit test (Mock requests)
2. Integration tests (with real API keys and calls)

But this work is a bit bigger piece of work. Need prioritisation :)

Reply:
So this is what Im suggesting:

Keep you most recent change for the `Test the Util` section
```
## Test The Util
Go to `tests/static_data` directory and create a file called "API_KEY" and paste your API Key in this file. 
```

And lets add a new section called `SDK Usage (quickdemo, quick start guide, you can pick whatever makes more sense)`
```
## SDK Usage
This is a quick sample of how to run your current Python app with our SDK:
(old sample code would go here)
from fattmerchant.client import FMClient	
fatt = FMClient(	
    "API_KEY_HERE",	
    "local"	
)
 ... and so on.
```

Reply:
ok, SDK Usage makes sense. One more option to look at is creating an example folder and creating a basic_client.py and advance_client.py type of classes. 

for now I will just create "SDK Usage"